README.md

```diff
@@ -50,42 +50,7 @@ pip install astro-sdk-python[amazon,google,snowflake,postgres]
 
 1. Copy the following DAG into a file named `calculate_popular_movies.py` and add it to the `dags` directory of your Airflow project:
 
-    ```Python
-    from datetime import datetime
-    from airflow import DAG
-    from astro import sql as aql
-    from astro.files import File
-    from astro.sql.table import Table
-
-    @aql.transform()
```

Comment:
If we add this back to the README as code, it would be great to add some more inline comments about what the code/ SDK is doing here.